libs/prebuilt/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -649,6 +669,15 @@ def should_continue(state: StateSchema) -> Union[str, list]:
     # This means that this node is the first one called
     workflow.set_entry_point(entrypoint)
 
+    agent_paths = ["tools", END]
+    post_model_hook_paths = ["tools", END]
+
+    # Add a post model hook node if post_model_hook is provided
+    if post_model_hook is not None:
+        workflow.add_node("post_model_hook", post_model_hook)
+        workflow.add_edge("agent", "post_model_hook")
```

Comment:
i think this should be handled by the `should_continue` - i think we should remove